README.md

```diff
@@ -8,10 +8,13 @@ together.
 
 ## Headers
 
-TODO: explain why two headers here.
+Trace context represented by set of name/value pairs describing identity of every http request. As a performance optimization measure few of these pairs were promoted to a separate header. This header has fixed length and defined sequence of fields.
 
-* See [TraceContext Header](trace_context/README.md)
-* See [CorrelationContext Header](correlation_context/README.md)
+Libraries and platforms SHOULD propagate `Trace-Context` and `Trace-Context-Ext` headers to guarantee that trace will not be broken. `Correlation-Context` header is a companion header representing user-defined baggage associated with the trace.
```

Comment:
SHOULD to MUST I think? Otherwise we are specifying something one can meet without being compatible with anything, right?

Reply:
I think the trickiness is in the form here.. ex A library must propagate an incoming trace-context with any trace-context-ext. If not, it must restart or abandon the trace. (ack this may take time to elaborate, but in such a case adding a TODO to this doc with a link is also fine).